docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -194,10 +195,6 @@ public String config() throws IOException, InterruptedException {
     @Override
     public boolean writeLogs(String container, OutputStream output) throws IOException {
         try {
-            Awaitility.await()
-                    .pollInterval(50, TimeUnit.MILLISECONDS)
-                    .atMost(LOG_WAIT_TIMEOUT.getMillis(), TimeUnit.MILLISECONDS)
-                    .until(() -> exists(container));
```

Comment:
why don't we wait for the container anymore?
If we later change the position of collectLogs then this might still break

Reply:
Ah so because we were waiting on all the containers, once they were all up (and we'd see the "docker-compose cluster started" log message), a container would go down in the interim, and then it'd time out here?

I think we should explicitly handle the case where a container _we_ started (and are tracking) went down, and fail fast in that case (while still collecting the logs).